.github/workflows/PR-CI.yml

```diff
@@ -95,4 +95,5 @@ jobs:
       # has to be scaffolded outside the CLI project so that no lint/tsconfig are leaking
       # through. this way it ensures that it is the app's configs that are being used
       - run: pnpm start -y ../ci-test-app
+      - run: mv ../ci-test-app/.env-example ../ci-test-app/.env
```

Comment:
oh I like this approach. this way we still leave it up to the user to make sure they set their env-vars whilst allowing the CI to test-build the app 👍🏼